### DISCUSSION TITLE
Enabling alpha matting degrades image quality

### DISCUSSION THREAD

**Discussion by chameroy** [danielgatis/rembg#165] [Q&A]
Hello, whenever I enable alpha matting (using -a -ae 15), the resulting image always comes out blurry. When I don't enable alpha matting, the image appears to be as clear as the original image, but there's erosion damage around the edge.

What kinds of images are candidates for alpha matting or should it work for all of them?

**Comment by renato-bosa** [comment]
Are this alpha parameters u2net parameters, or is it a rembg post-processing thing?

Thanks.

**Reply by renato-bosa** [reply]
According to what I saw on bg.py, it looks like it's a rembg post-processing thing.
Can you confirm it @danielgatis ?
